homeassistant/components/homeassistant/scene.py

```diff
@@ -129,6 +136,18 @@ def _convert_states(states):
         vol.Schema({vol.Required(CONF_ENTITIES): STATES_SCHEMA}),
     )
 
+    async def create_service(call):
+        """Create a scene."""
+        async_add_entities(
```

Comment:
Unfortunately, this is not working as expected. The scenes are not there after this call. What am I missing?

Reply:
@balloob Any ideas here?

Reply:
nvm `async_add_entities` needs a list